### DISCUSSION TITLE
Home page doesn't load post power failure

### DISCUSSION THREAD

**Discussion by svetlinski** [netbox-community/netbox#9784] [Q&A] [answered]
### NetBox version

v3.1.6

### Python version

3.8

### Steps to Reproduce

Open the home page

### Expected Behavior

Home page to load without issues

### Observed Behavior

Had a power outage recently, since server is back, I am unable to get to the main page at all. 
Page doesn't load, displays an error about CSS files:
![image](https://user-images.githubusercontent.com/99681269/179854435-7c01f268-5ac0-482d-90fd-d9ddeefd8116.png)
I have tried the below:
Changing the nginx user to www-data; changing permissions for the media and static folders to 775, 777 for the www-data user; restarting nginx, netbox and netbox-rq processess
Changing the nginx user to root; changing permissions for the media and static folders to 775, 777 for the www-data user; restarting nginx, netbox and netbox-rq processess
Changing the nginx user to netbox; changing permissions for the media and static folders to 775, 777 for the www-data user; restarting nginx, netbox and netbox-rq processess
I have also tried restoring 2 previous backups, it makes no difference at all. 
I read all the available discussions online and they didn't help, especially not the one about chown of the folders. Now, I am stuck for any other ideas.
Any help is more than welcome.
Thanks in advance.

**Comment by drmsoffall** [comment]
Not a lot to go on here, but if the server was powered on continually (and never rebooted) since you installed NetBox, check settings that might not persist through a reboot/power loss.  For example, this could be a SELinux issue.  You could test that theory by doing a `setenforce 0` ... though if that works, I suggest doing a proper, persistent SELinux configuration rather than leaving it disabled. 😃 

In any case, I suggest checking your nginx error logs for the exact error message when you try to load the page.

**Reply by svetlinski** [reply]
The strange thing is that the server has been rebooted many times before, for one reason or another. It seems that this time, something else has happened that has caused this. Previously, restarts have been controlled through server CLI or GUI. This time, the power failure meant there was no control over the shutdown. 

I will check the nginx error logs now.

**Comment by svetlinski** [comment, marked as answer]
I hope this helps someone else in the same situation, but checking the nginx logs helped me:
2022/07/20 09:05:33 [error] 117920#117920: *39 open() "**/opt/netbox/netbox-3.1.6/static/netbox-external.css**" failed (2: No such file or directory), client: ::ffff:192.x.x.x, server: netbox.example.com, request: "GET /static/netbox-external.css?v=3.1.6 HTTP/1.1", host: "192.x.x.x", referrer: "https://192.x.x.x/"

The file is actually located in **/opt/netbox/netbox-3.1.6/netbox/static/**. 

To fix this, edit the /etc/nginx/sites-available/netbox file:
 location /static/ {
        alias /opt/netbox/netbox-3.1.6**/netbox/**static/;
